main/kind.h

```diff
@@ -0,0 +1,37 @@
+/*
+*   Copyright (c) 1998-2003, Darren Hiebert
```

Comment:
The definition of kindOption is just moved from parse.h. So I keep the Copyright notice.